kong/cli/services/serf.lua

```diff
@@ -28,7 +28,11 @@ function Serf:_get_cmd()
   local cmd, err = Serf.super._get_cmd(self, {}, function(path)
     local res, code = IO.os_execute(path.." version")
     if code == 0 then
-      return res:match("^Serf v0.7.0")
+      local version_match = res:match("^Serf v0.7.0")
+      if not version_match then
+        logger:warn("Incompatible Serf version in the path. Required version is Serf v0.7.0")
+      end
+      return version_match
```

Comment:
Since we still execute the rest of the script here and if I compare this change with what you reported in #946, am I right to assume the output will now be this:

```
[WARN] Incompatible Serf version in the path. Required version is Serf v0.7.0
[ERR] Can't find serf
[ERR] Could not start Kong
```

If so, we should probably change the warning to be an actual error and exit the CLI, don't we?

PS: also thinking we should not actually require `0.7.0`, but `~0.7.0` or even `>=0.7.0`, but this is unrelated to this PR.